### ISSUE TITLE
unable to import component

### ISSUE THREAD

**Issue by flyfj** [rhostem/ng2-simple-autocomplete#1]
when i put the component into ngmodule declarations, i got an error saying 'Unexpected value 'Ng2SimpleAutocomplete' declared by the module 'AppModule'. Please add a @Pipe/@Directive/@Component annotation.'

I'm using the latest version of angular, any idea how this could happen?

**Comment by rhostem** [comment]
@flyfj I think you might have added Ng2SimpleAutocomplete  at root NgModule that named AppModule.... Sorry I have no idea which problem makes that message. 

I just tried this autocomplete module with [angular-starter](https://github.com/AngularClass/angular-starter) which uses angular@4.3.2 and it works well. 'angular starter' uses webpack, so I tested on my own private project which uses angular cli@1.0.0, angular@4.0.0. It works well too.

And it may be not related to your problem, I figured out this package requires 'tslib' while testing. So I moved it to the dependencies list from devDependencies and published a new version.
